tests/mypy/outputs/plugin-success.txt

```diff
@@ -0,0 +1,3 @@
+121: error: Unexpected keyword argument "name" for "AddProject"  [call-arg]
```

Comment:
surely this file should not exist. As per the name, mypy should pass on this.

Reply:
I deleted this file. And, I added these error messages on `fail4.txt` which is used by only mypy test case without the plugin.